### ISSUE TITLE
Show date in content view

### ISSUE THREAD

**Issue by inkhey** [tracim/tracim#2412]
## Bug description & symptom

It may be nice to know which file are lastly updated directly in content list view

## Diagnostic

In backend, we do have 2 date: 
- updated : last update of content
- created : creation of content date.

It's seems that updated is needed to be visible easily but created can also be nice to have with a mouse over or something similar.

## Expected

to be discuss.

**Comment by celiamgtt** [comment]
I agree, it could be really interesting to have an indication of time directly in the list of contents view.
